### DISCUSSION TITLE
How to make event message on the window's titleBar close Button

### DISCUSSION THREAD

**Discussion by Jlim01** [lepoco/wpfui#1111] [Q&A]
WPF.UI support title bar. and there is a close button, maxmize, minimize button.
I wanna implemented close button,  when the user click close button, then once show message box like "do you wanna close application?" , if they click yes then, it's app is completely exit. 
is there this function? I tried access to Close keyword's method. but I couldn't control because the App's UI first of all exit, and then accessed the method related to Close keyword.

I found             ui:TitleBar.CloseClicked="TitleBar_CloseClicked"  this ui's clicked event. but I think it is useless implemented what i wanna.
finally App is exit

**Comment by Veritera** [comment]
Right now this is not possible with the current implementation.
